mindsdb/integrations/handlers/llama_index_handler/github_loader_helper.py

```diff
@@ -0,0 +1,65 @@
+import os
+from llama_hub.github_repo import GithubRepositoryReader
+
+
+def _get_github_token(args, connection_args):
```

Comment:
Is it possible to implement this using the `get_api_key()` method from `mindsdb.integrations.utilities.handler_utils`?

Reply:
I don't know, it is similar to the _get_open_ai key from the `llama_index_handler.py`.

Reply:
Hmm.. If possible, let's replace the use of both with this method. It will make our code a little more DRY.